x-pack/plugins/apm/public/components/app/service_inventory/service_list/index.tsx

```diff
@@ -260,10 +267,13 @@ export function ServiceList({
       gutterSize="xs"
       direction="column"
       responsive={false}
-      alignItems="flexEnd"
+      // alignItems="flexEnd"
     >
       <EuiFlexItem>
         <EuiFlexGroup responsive={false} alignItems="center" gutterSize="xs">
+          <EuiFlexItem>
+            {fallbackToTransactions && <AggregatedTransactionsBadge />}
+          </EuiFlexItem>
```

Comment:
You could check the fallbackToTransactions and then add the `EuiFlexItem`
```suggestion
{fallbackToTransactions && 
          <EuiFlexItem>
              <AggregatedTransactionsBadge />
          </EuiFlexItem>
}
```